src/datepicker/datepicker-input.spec.ts

```diff
@@ -160,6 +160,10 @@ describe('NgbInputDatepicker', () => {
          expect(input.disabled).toBeTruthy();
          expect(buttonInDatePicker.disabled).toBeTruthy();
 
+         // check if ngb-datepicker popup is disabled
+         const navItem = fixture.nativeElement.querySelector('ngb-datepicker-navigation');
+         expect(navItem.getAttribute('ng-reflect-disabled')).toBeTruthy();
```

Comment:
`ng-reflect-*` attributes are Angular internal implementation detail and we should never ever use this info in our implementation / tests.